src/components/ShortcutsDialog.tsx

```diff
@@ -280,6 +280,22 @@ export const ShortcutsDialog = ({ onClose }: { onClose?: () => void }) => {
                 label={t("labels.bringForward")}
                 shortcuts={[getShortcutKey("CtrlOrCmd+]")]}
               />
+              <Shortcut
+                label={t("labels.alignTop")}
+                shortcuts={[getShortcutKey("CtrlOrCmd+Shift+🠝")]}
+              />
+              <Shortcut
+                label={t("labels.alignBottom")}
+                shortcuts={[getShortcutKey("CtrlOrCmd+Shift+🠟")]}
+              />
+              <Shortcut
+                label={t("labels.alignLeft")}
+                shortcuts={[getShortcutKey("CtrlOrCmd+Shift+🠜")]}
+              />
+              <Shortcut
+                label={t("labels.alignRight")}
+                shortcuts={[getShortcutKey("CtrlOrCmd+Shift+🠞")]}
+              />
```

Comment:
I'd use regular words here (Left, Right...).

Reply:
Yeah, good idea. Changed that already.